raiden_contracts/tests/test_channel_cooperative_settle.py

```diff
@@ -530,5 +530,10 @@ def test_cooperative_settle_channel_event(
         signature_A
     )
 
-    ev_handler.add(txn_hash, E_CHANNEL_SETTLED, check_channel_settled(channel_identifier))
+    ev_handler.add(txn_hash, E_CHANNEL_SETTLED, check_channel_settled(
+            channel_identifier,
+            balance_B,
+            balance_A
+            )
+        )
```

Comment:
As flake8 complains, this can be written like this:
```
ev_handler.add(txn_hash, E_CHANNEL_SETTLED, check_channel_settled(
        channel_identifier,
        balance_B,
        balance_A
))
```